clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/AutoScalingWorker.groovy

```diff
@@ -387,23 +387,40 @@ class AutoScalingWorker {
    * This is used to gradually roll out launch template.
    */
   private boolean shouldSetLaunchTemplate() {
+    // Request level flag that forces launch configurations.
     if (!setLaunchTemplate) {
       return false
     }
 
+    // Property flag to turn off launch template feature. Caching agent might require bouncing the java process
     if (!dynamicConfigService.isEnabled("aws.features.launch-templates", false)) {
       log.debug("Launch Template feature disabled via configuration.")
       return false
     }
 
-    // application allow list: app1,app2
+    // This is a comma separated list of applications to exclude
+    String excludedApps = dynamicConfigService
+      .getConfig(String.class, "aws.features.launch-templates.excluded-applications", "")
+    if (application in excludedApps.split(",")) {
+      return false
+    }
+
+    // Application allow list with the following format:
+    // app1:account:region1,region2,app2:account:region1
```

Comment:
ex: `gate:mgmt:us-east-1,keel:mgmt:us-west-2,eu-west-1....`

Reply:
Can this accept `app:account` with no region?

Reply:
it requires a one or more regions. Can enumerate all regions in there to achieve the same result.